.github/workflows/build.yaml

```
@@ -0,0 +1,72 @@
+on:
+  # to trigger build when daily tinytex build is finished
+  repository_dispatch:
+    types: [daily-build]
+  workflow_dispatch:
+
+    
+name: Build Daily TinyTeX Full Bundles
+
+env:
+  TINYTEX_INSTALLER: TinyTeX
+  GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GH_REPO: cderv/tinytex-releases
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+    name: Build Bundles For Linux
+
+    steps:
+      - name: Clone repo
+        uses: actions/checkout@v3
+
+      - name: Build TeX Live full
+        run: |
+          wget https://yihui.org/tinytex/TinyTeX.tar.gz
+          tar xzf TinyTeX.tar.gz
+          ./.TinyTeX/bin/*/tlmgr install scheme-full
+          tar zcf TinyTeX-2.tar.gz .TinyTeX
+          ls -lh TinyTeX*
+          rm -r .TinyTeX TinyTeX.tar.gz
+
+      - name: update daily release
+        run: |
+          gh release upload daily TinyTeX-2.tar.gz --clobber
+
+  build-windows:
+    runs-on: windows-latest
+    name: Build Bundles For Windows
+
+    steps:
+    - name: Clone repo
+      uses: actions/checkout@v3    
+
+    - name: Build Full TinyTeX
+      run: |
+        [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; Invoke-WebRequest https://yihui.org/tinytex/install-bin-windows.bat -OutFile install-bin-windows.bat
```

Why not simply use `curl` instead of this Powershell command?
```suggestion
        curl -OL https://yihui.org/tinytex/install-bin-windows.bat
```

---

Using pure powershell was only to be able to use the line even if no curl available. In this new context I agree we could change.

Initially we did not use curl because appveyor may not have it maybe. Here I copy pasted from old script.
If `curl` is available by default in `windows-latest` then yes we could use it.

---

> If curl is available by default in windows-latest then yes we could use it.

Curl is available by default on any Windows machine nowadays, so obviously it should be there on the default machines too.